sentry/tests/test_tracing.rs

```diff
@@ -65,23 +65,32 @@ fn test_span_record() {
         .with(sentry_tracing::layer())
         .set_default();
 
-    let options = sentry::ClientOptions{
+    let options = sentry::ClientOptions {
         traces_sample_rate: 1.0,
         ..Default::default()
     };
 
-    let envelopes = sentry::test::with_captured_envelopes_options(|| {
-        let _span = tracing::span!(tracing::Level::INFO, "span").entered();
-        function();
-    }, options);
+    let envelopes = sentry::test::with_captured_envelopes_options(
+        || {
+            let _span = tracing::span!(tracing::Level::INFO, "span").entered();
+            function();
+        },
+        options,
+    );
 
     assert_eq!(envelopes.len(), 1);
 
     let envelope_item = envelopes[0].items().next().unwrap();
     let ref transaction = match envelope_item {
         sentry::protocol::EnvelopeItem::Transaction(t) => t,
-        _ => { assert!(false); return; }
+        _ => {
+            assert!(false);
```

Comment:
Since this is test code, you can as well `panic!("expected only a transaction item");`
As the panic diverges, you also don’t need the return/block afterwards.

Reply:
Thanks. I'll know